### PR TITLE
charts,manifests: Update documentation links in OCP OLM packages to use docs.openshift.com

### PR DESCRIPTION
**Pull request by chancez**
/hold
/assign @lamek

### CODE REVIEW

File: charts/metering-ansible-operator/upstream-values.yaml [hunk 1]
```diff
@@ -49,6 +49,41 @@ olm:
...
+
+      * **HTTP API** - Reports can be queried from an in-cluster HTTP API in addition to reading from persistent storage.
+
+      ### Before you start
+
+      Metering runs a big data stack to crunch your data and requires at least 4GB of RAM and 4 CPU cores. At least one Node should have 2GB of RAM and 2 CPU cores. Memory and CPU consumption may often be lower, but will spike when running reports, or collecting data for larger clusters.
+
+      ### Common Configurations
+
+      Metering works out of the box without any customizations or configuration. Read the [full documentation](https://github.com/operator-framework/operator-metering/blob/master/Documentation/metering-config.md) for more details.
```
Comment (lamek) [inline charts/metering-ansible-operator/upstream-values.yaml]: Do you want to still link to GH here? If not, I'd link to About Metering:
https://docs.openshift.com/container-platform/4.2/metering/metering-about-metering.html

File: charts/metering-ansible-operator/values.yaml [hunk 2]
```diff
@@ -222,17 +221,20 @@ olm:
 
       ### Before you start
 
-      Metering runs a big data stack to crunch your data and requires at least 4GB of RAM and 4 CPU cores. At least one Node should have 2GB of RAM and 2 CPU cores. Memory and CPU consumption may often be lower, but will spike when running reports, or collecting data for larger clusters.
+      Metering runs a big data stack to crunch your data and requires at least 4GB of RAM and 4 CPU cores.
+      At least one Node should have 2GB of RAM and 2 CPU cores.
+      Memory and CPU consumption may often be lower, but will spike when running reports, or collecting data for larger clusters.
+
+      Metering requires configuring storage, pleaese review the [configuring persistent storage documentation](https://docs.openshift.com/container-platform/4.2/metering/metering-configure-persistent-storage.html) before proceeding.
```
Comment (lamek) [inline charts/metering-ansible-operator/values.yaml]: Nit. Metering requires configuring storage. Review the [documentation on configuring persistent storage](https://docs.openshift.com/container-platform/4.2/metering/metering-configure-persistent-storage.html) before proceeding.

File: charts/metering-ansible-operator/values.yaml [hunk 3]
```diff
@@ -222,17 +221,20 @@ olm:
...
 
       ### Common Configurations
 
-      Metering works out of the box without any customizations or configuration. Read the [full documentation](https://github.com/operator-framework/operator-metering/blob/master/Documentation/metering-config.md) for more details.
+      * **Store data in object storage or in a PersistentVolume** - Store your report output [in an object storage bucket or in a PersistentVolume](https://docs.openshift.com/container-platform/4.2/metering/metering-configure-persistent-storage.html).
```
Comment (lamek) [inline charts/metering-ansible-operator/values.yaml]: This link is one level down:
https://docs.openshift.com/container-platform/4.2/metering/configuring-metering/metering-configure-persistent-storage.html

File: manifests/deploy/ocp-testing/olm/bundle/4.2/meteringoperator.v4.2.0.clusterserviceversion.yaml [hunk 4]
```diff
@@ -233,17 +233,19 @@ spec:
 
     ### Before you start
 
-    Metering runs a big data stack to crunch your data and requires at least 4GB of RAM and 4 CPU cores. At least one Node should have 2GB of RAM and 2 CPU cores. Memory and CPU consumption may often be lower, but will spike when running reports, or collecting data for larger clusters.
+    Metering runs a big data stack to crunch your data and requires at least 4GB of RAM and 4 CPU cores.
+    At least one Node should have 2GB of RAM and 2 CPU cores.
+    Memory and CPU consumption may often be lower, but will spike when running reports, or collecting data for larger clusters.
 
-    ### Common Configurations
+    Metering requires configuring storage, pleaese review the [configuring persistent storage documentation](https://docs.openshift.com/container-platform/4.2/metering/metering-configure-persistent-storage.html) before proceeding.
```
Comment (lamek) [inline manifests/deploy/ocp-testing/olm/bundle/4.2/meteringoperator.v4.2.0.clusterserviceversion.yaml]: https://docs.openshift.com/container-platform/4.2/metering/configuring-metering/metering-configure-persistent-storage.html

File: manifests/deploy/ocp-testing/olm/bundle/4.2/meteringoperator.v4.2.0.clusterserviceversion.yaml [hunk 5]
```diff
@@ -233,17 +233,19 @@ spec:
...
 
-    Metering works out of the box without any customizations or configuration. Read the [full documentation](https://github.com/operator-framework/operator-metering/blob/master/Documentation/metering-config.md) for more details.
+    ### Common Configurations
 
-    * **Use a specific StorageClass** - Follow the example to [use your own StorageClass](https://github.com/operator-framework/operator-metering/blob/master/manifests/metering-config/shared-storage.yaml) instead of the Dynamic Provisioner.
+    * **Store data in object storage or in a PersistentVolume** - Store your report output [in an object storage bucket or in a PersistentVolume](https://docs.openshift.com/container-platform/4.2/metering/metering-configure-persistent-storage.html).
```
Comment (lamek) [inline manifests/deploy/ocp-testing/olm/bundle/4.2/meteringoperator.v4.2.0.clusterserviceversion.yaml]: https://docs.openshift.com/container-platform/4.2/metering/configuring-metering/metering-configure-persistent-storage.html

File: manifests/deploy/openshift/olm/bundle/4.2/meteringoperator.v4.2.0.clusterserviceversion.yaml [hunk 6]
```diff
@@ -233,17 +233,19 @@ spec:
 
     ### Before you start
 
-    Metering runs a big data stack to crunch your data and requires at least 4GB of RAM and 4 CPU cores. At least one Node should have 2GB of RAM and 2 CPU cores. Memory and CPU consumption may often be lower, but will spike when running reports, or collecting data for larger clusters.
+    Metering runs a big data stack to crunch your data and requires at least 4GB of RAM and 4 CPU cores.
+    At least one Node should have 2GB of RAM and 2 CPU cores.
+    Memory and CPU consumption may often be lower, but will spike when running reports, or collecting data for larger clusters.
 
-    ### Common Configurations
+    Metering requires configuring storage, pleaese review the [configuring persistent storage documentation](https://docs.openshift.com/container-platform/4.2/metering/metering-configure-persistent-storage.html) before proceeding.
```
Comment (lamek) [inline manifests/deploy/openshift/olm/bundle/4.2/meteringoperator.v4.2.0.clusterserviceversion.yaml]: https://docs.openshift.com/container-platform/4.2/metering/configuring-metering/metering-configure-persiste

File: manifests/deploy/openshift/olm/bundle/4.2/meteringoperator.v4.2.0.clusterserviceversion.yaml [hunk 7]
```diff
@@ -233,17 +233,19 @@ spec:
...
 
-    Metering works out of the box without any customizations or configuration. Read the [full documentation](https://github.com/operator-framework/operator-metering/blob/master/Documentation/metering-config.md) for more details.
+    ### Common Configurations
 
-    * **Use a specific StorageClass** - Follow the example to [use your own StorageClass](https://github.com/operator-framework/operator-metering/blob/master/manifests/metering-config/shared-storage.yaml) instead of the Dynamic Provisioner.
+    * **Store data in object storage or in a PersistentVolume** - Store your report output [in an object storage bucket or in a PersistentVolume](https://docs.openshift.com/container-platform/4.2/metering/metering-configure-persistent-storage.html).
```
Comment (lamek) [inline manifests/deploy/openshift/olm/bundle/4.2/meteringoperator.v4.2.0.clusterserviceversion.yaml]: https://docs.openshift.com/container-platform/4.2/metering/configuring-metering/metering-configure-persiste

File: charts/metering-ansible-operator/upstream-values.yaml (hunk 1 above)
Comment (lamek) [reply charts/metering-ansible-operator/upstream-values.yaml]: nvm, I see this is the upstream values.

**Comment by chancez** [comment]
@lamek Updated

**Pull request merged**